.github/workflows/integration.yml

```
@@ -101,6 +101,13 @@ jobs:
           repository: "${{ needs.configure.outputs.repo-name }}"
           persist-credentials: false
 
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
```

```suggestion
          key: ${{ runner.os }}-${{ matrix.component }}-buildx-${{ github.sha }}
```

I would also index by component, to avoid issues due to concurrency and prevent deleting the cache of one component when another is built (in the move cache step)